### ISSUE TITLE
curl: (60) SSL certificate problem

### ISSUE THREAD

**Issue by jacobrosenthal** [balena-io-projects/balena-rust-hello-world#23]
```
[backend]  curl: (60) SSL certificate problem: unable to get local issuer certificate
[backend]  More details here: https://curl.haxx.se/docs/sslcerts.html
[backend]  curl failed to verify the legitimacy of the server and therefore could not
[backend]  establish a secure connection to it. To learn more about this situation and
[backend]  how to fix it, please visit the web page mentioned above.
```

Related forum
https://forums.balena.io/t/unable-to-complete-the-sense-application-help-please/19698/6

**Comment by zrzka** [comment]
Hi @jacobrosenthal, thanks for the issue and for the PR. Can you please retest with `master` now?

See [this commit](https://github.com/balena-io-projects/balena-rust-hello-world/commit/d7e646bcb3d7509336c80c13015c356894597125). It fixes the issue for me without replacing `wget` with `curl`. I slightly prefer this way, because it's a verbatim copy & paste from the [rustup.rs](https://rustup.rs) site.

**Comment by jacobrosenthal** [comment]
Yes this seems to work! The forums should be notified of the rehash fix as the official recommendation there is wget usage.

**Comment by jacobrosenthal** [comment]
Fixed in #22

**Comment by zrzka** [comment]
Thanks for testing @jacobrosenthal!
